packages/nimble-components/src/wafer-map/specs/features/rendering.md

```diff
@@ -82,6 +82,48 @@ The POC is found in this branch [Worker Rendering POC](https://github.com/ni/nim
 
 ### Data Structure and Interface
 
+The best solution to solve the API of the wafermap is to use both of the proposed methods.
```

Comment:
Can you provide a brief summary of what "both of the proposed methods" is referring to? Being in a new section it's unclear at a glance what the referenced methods are.